src/main/kotlin/view/FilterPanel.kt

```diff
@@ -6,12 +6,21 @@ import javafx.geometry.Insets
 import javafx.geometry.Orientation
 import javafx.geometry.Pos
 import javafx.geometry.Side
+import javafx.scene.chart.AreaChart
+import javafx.scene.chart.NumberAxis
+import javafx.scene.chart.XYChart
+import javafx.scene.control.ComboBox
+import javafx.scene.control.ScrollPane
+import javafx.scene.control.Slider
 import javafx.scene.control.TabPane
 import javafx.scene.text.FontWeight
 import models.EngineModel
-import processing.filters.BlurType
-import processing.filters.HSVType
-import processing.filters.RGBType
+import processing.BlurType
+import processing.HSVType
+import processing.RGBType
```

Comment:
ubunto running test gives the following error: 
/home/runner/work/IPEwG/IPEwG/src/main/kotlin/view/FilterPanel.kt: (45, 71): Unresolved reference: RGBType

Reply:
Will do